code_schemes/yes_no.json

```diff
@@ -0,0 +1,92 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Yes_No",
```

Comment:
For yes/no questions, probably best to include some hint as to what these are in response to. It's normal to have multiple schemes with yes/no in a project, and we need to be able to tell them apart

Reply:
Will include a hint on the Name. Thanks @as2388!